### PR TITLE
Support backup/restore of all tables (*) and data only restore mode.

### PR DESCRIPTION
**Pull request by rstml**
Added option to backup/restore all tables (* wildcard) and restore data only (useful for batch loading data). Default write capacity is now 25 to stay within free tier.

### CODE REVIEW

File: dynamodump.py [hunk 1]
```diff
@@ -388,7 +396,8 @@ def do_restore(conn, sleep_interval, source_table, destination_table, write_capa
 
   if dest_table.find("*") != -1:
     matching_destination_tables = get_table_name_matches(conn, dest_table, prefix_separator)
-    logging.info("Found " + str(len(matching_destination_tables)) + " table(s) in DynamoDB host to be deleted: " + ", ".join(matching_destination_tables))
+    deleteStr = ": " if args.dataOnly else " to be deleted: "
```
Comment (bchew) [inline dynamodump.py]: Minor one, but could you please change this to `delete_str` to match the convention of the rest of the file?

File: dynamodump.py [hunk 2]
```diff
@@ -388,7 +396,8 @@ def do_restore(conn, sleep_interval, source_table, destination_table, write_capa
...
+    logging.info("Found " + str(len(matching_destination_tables)) + " table(s) in DynamoDB host" + deleteStr + ", ".join(matching_destination_tables))
```
Comment (bchew) [inline dynamodump.py]: Here too

**Comment by bchew** [comment]
@rstml thanks for opening a PR! Have gone through your changes and have a minor formatting change to request.

Would merge it in after, thanks!

**Comment by rstml** [comment]
ready

**Comment by bchew** [comment]
Thanks! Merging

**Pull request merged**